content/en/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -401,17 +433,22 @@ mounted to the appropriate mount points.
 
 ## Scaling a StatefulSet
 
-Scaling a StatefulSet refers to increasing or decreasing the number of replicas.
+Scaling a StatefulSet refers to increasing or decreasing the number of replicas
+(horizontal scaling).
 This is accomplished by updating the `replicas` field. You can use either
 [`kubectl scale`](/docs/reference/generated/kubectl/kubectl-commands/#scale) or
 [`kubectl patch`](/docs/reference/generated/kubectl/kubectl-commands/#patch) to scale a StatefulSet.
 
-### Scaling Up
+### Scaling up
+
+Scaling up means adding more replicas. If you balance work across the
+StatefulSet, the new larger set of Pods can perform more of it.
```

Comment:
"If you .... of it".
This sentence is not clarifying any thing.
It in fact introduce some confusion to users, i.e. how am I supposed to balance work across a SatefuleSet? Why do we call that larger set a new set?
Yes, the intent to add more replicas is crystal clear, isn't it?

Reply:
(I tried that new text)